frontend/src/entities/group/group.columns.ts

```diff
@@ -34,9 +34,15 @@ export const groupColumns: TableProps.ColumnDefinition<IGroup>[] = [{
     header: 'Description',
     sortingField: 'description',
     cell: (item) => item.description,
+},
+{
+    id: 'members',
+    header: 'Members',
+    sortingField: 'members',
+    cell: (item) => item.numMembers || '-',
```

Comment:
I think we have a display function for this. I'll approve now so this can get in the push today but you should follow up on fixing this I think.